### ISSUE TITLE
Mehrere Reihenfolgen korrekt, sind aber nicht als korrekt markiert 

### ISSUE THREAD

**Issue by efinder2** [binsky08/amct#3]
In der Anforderungsreihenfolge, ist es nur wichtig, dass zu erst beides angefordert und zugeteilt ist. Daher ist dies "komplett" falsche Lösung in meinen Augen korrekt
<img width="400" alt="image" src="https://user-images.githubusercontent.com/12611194/180612971-7ede7515-eec3-49a4-abec-b64a6e5556da.png">

**Comment by binsky08** [comment]
Ja, das Problem ist, dass man zur Zeit nicht mehrere Kombinationen von Antwortmöglichkeiten konfigurieren kann. (Vielleicht fällt mir ja ein gutes Konzept ein das noch umzusetzen)

Ich habe daher versucht die Antworten durch eine detaillierte Aufgabenbeschreibung weitestgehend einzuschränken.
Grundsätzlich hast du natürlich Recht. 

Immerhin regt die Aufgabe somit zum Nachdenken an xD
